decentraland/unity-explorer
# fix: add scene shader cut-out transparency

## What does this PR change?

Fixes #584 

Palm trees should look correct now

![image](https://github.com/decentraland/unity-explorer/assets/1999557/6c81c684-0503-4b9a-83ff-dc958dd770db)

![image](https://github.com/decentraland/unity-explorer/assets/1999557/b930f6be-7fe3-4cd7-9eb9-f96a17c03b4b)



## How to test the changes?



1. Launch the explorer
2. Check in the mall scene that the cut-out palms are nicely seen

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
